monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```diff
@@ -634,8 +634,7 @@ local addStyle(alias, colorMode, colors, dateFormat, decimals, mappingType, patt
         {"Non-Encrypted": "#E5AC0E"}, '', 'OSD Objectstore Types'
       )
       .addTarget(addTargetSchema('count(ceph_bluefs_wal_total_bytes)', 1, 'time_series', 'bluestore'))
-      .addTarget(addTargetSchema('count(ceph_osd_metadata) - count(ceph_bluefs_wal_total_bytes)', 1, 'time_series', 'filestore'))
-      .addTarget(addTargetSchema('absent(ceph_bluefs_wal_total_bytes)*count(ceph_osd_metadata)', 1, 'time_series', 'filestore')) + {gridPos: {x: 4, y: 8, w: 4, h: 8}},
+      .addTarget(addTargetSchema('count(ceph_osd_metadata) - count(ceph_bluefs_wal_total_bytes)', 1, 'time_series', 'filestore')) + {gridPos: {x: 4, y: 8, w: 4, h: 8}},
```

Comment:
There were 2 filestore counts? I removed one that wasn't correct.

Reply:
Are you sure? I have no clue about what this was for... :S Maybe @pcuzner recalls...

Reply:
@pcuzner ping

Reply:
@epuertat 
![image](https://user-images.githubusercontent.com/30913090/149942208-f2fc985c-0a64-44c2-87e2-34a4d799b857.png)
I deleted the wrong one :PPPPPP . In `ceph-dev` with 5 filestore osds you can see the output of both of the queries.

Reply:
_fixed in last commit_